src/awrtc/network/WebRtcNetwork.ts

```diff
@@ -205,10 +205,11 @@ export class WebRtcNetwork implements IBasicNetwork {
                 this.mSignalingNetwork.SendData(new ConnectionId(+key), buffer, true);
             }
 
-            if (peer.GetState() == WebRtcPeerState.Connected) {
+            // ignore peers in connected state which have been already added
+            if (peer.GetState() == WebRtcPeerState.Connected && !this.mIdToConnection[peer.ConnectionId.id]) {
```

Comment:
Now connection can be both _connected_ and _in signaling_ to handle adding new stream later.